lib/tasks/argo.rake

```diff
@@ -12,15 +12,19 @@ task :app_version do
 end
 
 def jettywrapper_load_config
-  return Jettywrapper.load_config.merge({:jetty_home => File.expand_path(File.dirname(__FILE__) + '../../../jetty'),:startup_wait => 200})
+  config = {
+    :jetty_home => File.expand_path(File.dirname(__FILE__) + '../../../jetty'),
+    :startup_wait => 240 # seconds - 4 mins
+  }
+  Jettywrapper.load_config.merge(config)
 end
 
 task :default => [:ci]
 
 task :ci do
-  ENV['RAILS_ENV'] = 'test'
+  raise 'NEVER RUN CI OUTSIDE RAILS_ENV=="test"' if ENV['RAILS_ENV'] != 'test'
```

Comment:
two minor suggestions:
1) i've seen rubocop and hound complain about using `do_stuff if some_thing != some_other_thing` and suggest `do_stuff unless some_thing == some_other_thing` instead.  maybe hound doesn't look at rake code by default?  anyway, i'd make the change just to silence future linter complaints, but this is not a thing that i personally feel strongly about.  but consistency is nice, i guess.
2) might be nice to log the actual env value (could just append `(RAILS_ENV is #{ENV["RAILS_ENV"]})` to the exception text, i think).